ghost-gcs-backup/templates/cronjob.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ template "ghost-gcs-backup.fullname" . }}
+  labels:
+    app: {{ template "ghost-gcs-backup.name" . }}
+    chart: {{ template "ghost-gcs-backup.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  concurrencyPolicy: Forbid
+  failedJobsHistoryLimit: 2
+  jobTemplate:
+    spec:
+      activeDeadlineSeconds: {{ default 60 .Values.job.activeDeadlineSeconds }}
```

Comment:
isn't this a small default for a backup? If not completed in a minute, job will be cancelled